client/deployment/nginx.conf

```diff
@@ -0,0 +1,21 @@
+events {
+    worker_connections 2000;
+}
+
+http {
+  server {
+    listen 9000;
+
+    location /graphql {
+      # This **needs** to use an environment variable for prod
```

Comment:
true, but probably the whole file would be replaced with configuration mounted as a volume?